.github/CODEOWNERS

```diff
@@ -181,3 +181,27 @@ README*                                         @ceph/doc-writers
 /src/test/cls_version                           @ceph/rgw
 /src/test/rgw                                   @ceph/rgw
 /src/test/test_rgw*                             @ceph/rgw
+
+# security (vulnerability-prone files/components)
+/sudoers.d/*                                    @ceph/security
+/src/ceph-crash.in                              @ceph/security  # CVE-2022-3650
+/src/auth/                                      @ceph/security  # CVE-2021-20288 CVE-2018-1128 CVE-2018-1129 CVE-2018-1128
+/src/init-ceph.in                               @ceph/security  # CVE-2013-1882
+/src/common/MemoryModel.cc                      @ceph/security  # CVE-2013-1882
+/src/msg/                                       @ceph/security  # CVE-2018-1128
+/src/mon/AuthMonitor.cc                         @ceph/security  # CVE-2021-20288
+/src/mon/Monitor.cc                             @ceph/security  # CVE-2016-5009
+/src/mon/OSDMonitor.cc                          @ceph/security  # CVE-2018-10861
+/src/rgw/rgw_acl_s3.cc                          @ceph/security  # CVE-2016-7031
+/src/rgw/rgw_asio_frontend.cc                   @ceph/security  # CVE-2020-1700 CVE-2019-10222
+/src/rgw/rgw_auth_s3.cc                         @ceph/security  # CVE-2018-16889
+/src/rgw/rgw_cors.cc                            @ceph/security  # CVE-2023-46159 CVE-2016-9579
+/src/rgw/rgw_op.cc                              @ceph/security  # CVE-2016-7031
+/src/rgw/rgw_policy_s3.cc                       @ceph/security  # CVE-2016-8626
+/src/rgw/rgw_rest_*.cc                          @ceph/security  # CVE-2023-43040 CVE-2021-3531
```

Comment:
some very common rgw sources here. are you sure you want to be spammed on every change to these?

i'm not sure this is going to be useful in practice. does someone on the security team actually plan to do these code reviews?

Reply:
Yes, you're right. In any case, I (we?) get spammed with hundreds of GH emails. We could instead add a Github label (`security`), to quickly identify and list PRs that modified files with a history of CVE/s. Another possibility would be to ensure that auth/input/header/validation code always happens in isolated files.

Thoughts?

Reply:
@epuertat i really appreciate the initiative you've taken here

while i personally don't think our development/review process is doing a poor job with respect to security, i'll certainly support improvements here

i'm far more concerned about our process for handling/fixing reported vulnerabilities. the switch to github's security advisories for CVEs is a big step in the right direction, and i want to see the necessary involvement from downstream projects to make this succeed

Reply:
👍🏼

Reply:
Would you please say more about what you expect Casey?

Reply:
@ktdreyer my expectation is that reported vulnerabilities be tracked and assigned in a timely manner. i won't go into details here, but i'll link you to an internal slack conversation

Reply:
Great, thanks.